templates/registration/login.html

```diff
@@ -102,6 +102,8 @@ <h2 data-i18n-key="outside-pages/login-header-title">Sign in to <span id="app-na
                                 <label>Password</label>
                                 <input type="password" name="password" placeholder="Password" />
                             </div>
+                            <input type="hidden" name="next" 
+                            value= {{ redirect_field_name }} />
```

Comment:
How did you determine the name `redirect_field_name`?

Reply:
the context param `next` is `REDIRECT_FIELD_NAME` obtained by this import `from django.contrib.auth import REDIRECT_FIELD_NAME` used in the LoginView, so I thought this name could be consistent. What do you think?